packages/developer-portal/src/sagas/current-member.ts

```diff
@@ -0,0 +1,75 @@
+import { put, fork, all, call, takeLatest, select } from '@redux-saga/core/effects'
+
+import { Action } from '@/types/core'
+import ActionTypes from '@/constants/action-types'
+import messages from '@/constants/messages'
+import { fetchCurrentMemberSuccess } from '@/actions/current-member'
+import { UpdateMemberModel } from '@reapit/foundations-ts-definitions'
+import { selectCurrentMemberData } from '@/selector/current-member'
+import { notification } from '@reapit/elements'
+import { fetchOrganisationMembers, updateOrganisationMemberById } from '@/services/developers'
+import { getDeveloperId, getLoggedUserEmail } from '@/utils/session'
+
+export const currentMemberFetch = function*() {
+  try {
+    const [developerId, currentMemberEmail] = yield all([call(getDeveloperId), call(getLoggedUserEmail)])
+    if (!developerId || !currentMemberEmail) {
+      throw new Error('Can not get developer ID or member email')
```

Comment:
this place I think it should have notification.error({}) instead of throw error

Reply:
I throw it so I can catch it below so that I only need one `notification.error` call